packages/CollabCodeEdit-Client.package/CCEModel.class/instance/initialize.st

```diff
@@ -1,11 +1,12 @@
 initialize-release
 initialize
 	super initialize.
-	self createSessionModel: CCECreateSessionModel new.
-	self account: CCEAccount default.
-	self joinSessionModel: CCEJoinSessionModel new.
-	self leaveSessionModel: CCELeaveSessionModel new.
-	self client: (CCEClient default owner: self).
-	self session: (CCESession default client: self client).
-	self session addStatusSubscriber: self.
-	self session imageAccount: self account.
+	self createSessionModel: CCECreateSessionModel new;
+		account: CCEAccount default;
+		joinSessionModel: CCEJoinSessionModel new;
+		leaveSessionModel: CCELeaveSessionModel new;
+		client: (CCEClient default owner: self);
+		session: (CCESession default client: self client).
+	self session 
+				addStatusSubscriber: self;
```

Comment:
This shows as some really weird indentation. It might be just GitHub, but please check for consistency.